functions.photon.php

```diff
@@ -271,3 +271,19 @@ function jetpack_photon_banned_domains( $skip, $image_url, $args, $scheme ) {
 
 	return $skip;
 }
+
+
+/**
+ * Jetpack Photon - Support Text Widgets.
+ * 
+ * @access public
+ * @param string $content Content from text widget.
+ * @return string
+ */
+function jetpack_photon_support_text_widgets( $content ) {
+    if ( class_exists( 'Jetpack_Photon' ) && Jetpack::is_module_active( 'photon' ) ) {
+        return Jetpack_Photon::filter_the_content( $content );
+    }
+
```

Comment:
That's correct. I got confused in last iteration. To clarify: the code block should be this:
```php
function jetpack_photon_support_text_widgets( $content ) {
	if ( class_exists( 'Jetpack_Photon' ) && Jetpack::is_module_active( 'photon' ) ) {
		return Jetpack_Photon::filter_the_content( $content );
	}
	return $content;
}
```
this is what I mean in my original comment when I said that you could just return instead of assign and then return.